Mathlib/Data/Finset/Defs.lean

```diff
@@ -119,22 +120,20 @@ instance decidableMem [_h : DecidableEq α] (a : α) (s : Finset α) : Decidable
 /-! ### set coercion -/
 
 /-- Convert a finset to a set in the natural way. -/
-@[coe] def toSet (s : Finset α) : Set α :=
-  { a | a ∈ s }
+instance : SetLike (Finset α) α :=
+  ⟨({a | a ∈ ·}), fun s₁ s₂ h ↦ (val_inj.symm.trans <| s₁.nodup.ext s₂.nodup).2 <| Set.ext_iff.mp h⟩
 
 /-- Convert a finset to a set in the natural way. -/
-instance : CoeTC (Finset α) (Set α) :=
-  ⟨toSet⟩
+abbrev toSet (s : Finset α) : Set α := s
 
-@[simp, norm_cast, grind =]
+@[norm_cast, grind =]
```

Comment:
did the simp tag here interfere with the one from SetLike?

Reply:
I think this becomes a special case of [SetLike.mem_coe](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/SetLike/Basic.html#SetLike.mem_coe); I'll check later.